orga/app/templates/components/routes/authenticated/campaigns/new-item.hbs

```diff
@@ -93,13 +93,7 @@
 
   <div class="form__field">
     <label for="custom-landing-page-text" class="label">Texte de la page d'accueil</label>
-    {{textarea id='custom-landing-page-text'
-            name='custom-landing-page-text'
-            maxlength='350'
-            rows=8
-            value=campaign.customLandingPageText
-            class="textarea"
-    }}
+    <Textarea @id="custom-landing-page-text" @name="custom-landing-page-text" @maxlength="350" @rows={{8}} @value={{campaign.customLandingPageText}} class="textarea" />
```

Comment:
`class` sans `@`

Reply:
Le codemod l'a pas changé du coup je sais pas ?

Reply:
D'après la [doc](https://guides.emberjs.com/release/components/built-in-components/#toc_textarea-), il n'y a effectivement pas besoin de `@` dans ce cas là (l'élément n'apparaissant pas, j'en déduie, qu'on peut l'utiliser normalement)

Reply:
Des infos intéressantes là aussi : https://guides.emberjs.com/release/components/component-arguments-and-html-attributes/#toc_html-attributes

> In addition, the class attribute is special, and will be merged with any existing classes on the element rather than overwriting them.